packages/trader/src/AppV2/Components/OrderDetails/__tests__/order-details.spec.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
```

Comment:
this test should pass without this import
```suggestion
```

Reply:
removed thanks